### PR TITLE
[ACM-11258]: Update kubevirt storage docs for HCP

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-11258

### CODE REVIEW

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 1]
```diff
@@ -30,6 +30,31 @@ hcp create cluster kubevirt \
...
+| `ReadWriteOnce` (RWO) `Block` or `Filesystem` RWX `Block` only
+| Supported
+| Use `Block` mode because `Filesystem` volume mode results in degraded guest `Block` mode performance. RWX `Block` volume mode is supported only when the hosted cluster is {ocp-short} 4.16 or later.
+
+| RWO `Block` storage
+| RWO `Block` storage or `Filesystem`
+| Not supported
+| Lack of live migration support affects the ability to update the underlying infrastructure cluster that hosts the KubeVirt VMs.
+
+| RWO `Block`
```
Comment (davidvossel) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: opps, i made a mistake in the table I put in jira.

This line (line 51) should read "RWO `FileSystem`".

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 2]
```diff
@@ -30,6 +30,31 @@ hcp create cluster kubevirt \
...
+| RWO `Block` or `Filesystem`
+| Not supported
+| Lack of live migration support affects the ability to update the underlying infrastructure cluster that hosts the KubeVirt VMs. The `Filesystem` volume mode results in degraded guest `Block` mode performance.
```
Comment (davidvossel) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: and one other area maybe we could add some clarity to. 

`The `Filesystem` volume mode results in degraded guest `Block` mode performance.`

might read more accurately

`The infrastructure  `Filesystem` volume mode results in degraded guest `Block` mode performance.`

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc (hunk 1 above)
Comment (lahinson) [reply clusters/hosted_control_planes/config_storage_kubevirt.adoc]: Updated, thanks!

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc (hunk 2 above)
Comment (lahinson) [reply clusters/hosted_control_planes/config_storage_kubevirt.adoc]: I revised line 54. PTAL and let me know if it's accurate.

**Comment by lahinson** [comment]
@davidvossel Thanks for reviewing the PR! I made a couple of changes based on your feedback. I also noticed that the term "guest cluster" had been introduced in this file by some other changes. I believe we decided a while back to use "hosted cluster" instead, so I updated instances of "guest" to "hosted".

**Review by davidvossel — COMMENTED** [review]
/lgtm

and good idea about changing the terminology from "guest" to "hosted" thanks!

**Review by jc-berger — APPROVED** [review]
/lgtn

**Pull request merged**